ptcg_cockatrice.xml

```diff
@@ -254,7 +254,7 @@
 			<pt>1/2</pt>
 			<tablerow>2</tablerow>
 			<text>2G: Evolve to Parasect
-				When another player loses life because of poison counters, you gain that much life.</text>
+				Whenever an opponent is dealt damage by a Pokémon they control with a poison counter on it, you gain that much life.</text>
```

Comment:
the weird thing about this effect is if you use explode like Electrode to deal damage to each player, and Electrode has a poison counter, you gain that much life.

Reply:
Yeah I thought about weird corner cases like that but I wasn't sure if it was an issue or not. Like a poisoned Pokemon turns all of their damage into poison damage so the effect still works? I wasn't sure of another way to make it work and I kinda like leaving room for unexpected interactions.